docs/charts/line.md

```diff
@@ -52,6 +52,13 @@ The line chart allows a number of properties to be specified for each dataset. T
 | [`borderWidth`](#line-styling) | `number` | Yes | - | `3`
 | [`cubicInterpolationMode`](#cubicinterpolationmode) | `string` | Yes | - | `'default'`
 | [`fill`](#line-styling) | <code>boolean&#124;string</code> | Yes | - | `true`
+| [`hoverBackgroundColor`](#line-styling) | [`Color`](../general/colors.md) | Yes | - | `undefined`
```

Comment:
Should we document `hoverCubicInterpolationMode` and `hoverFill`?

Reply:
Neither of those work. We are not going through `update` on hover, just updating the model with options and then `render`.
So bezier control points don't get updated. 
I did not investigate why `hoverFill` does not work.